pkg/lib/lockout/bucket.go

```diff
@@ -0,0 +1,76 @@
+package lockout
+
+import (
+	"strings"
+	"time"
+
+	"github.com/authgear/authgear-server/pkg/lib/config"
+)
+
+type BucketName string
+
+const (
+	BucketNameAccountAuthentication BucketName = "AccountAuthentication"
+)
+
+type BucketSpec struct {
```

Comment:
I think we can eliminate this struct and use AuthenticationLockoutConfig directly.

Or we rename this struct to LockoutSpec. Bucket is a term from rate limit. But I do not think we need to inherit this legacy in this module.